### PR TITLE
MHV-50831: Medications PDFs rich text and content updates

### PR DESCRIPTION
**Pull request by vmenshutin-bylight**
## Summary

Medications PDFs:

* Footer verbiage update
* Preface updated to support rich text
* Created ability for fields to support rich text
* Updated status definition field to rich text

## Related issue(s)

https://jira.devops.va.gov/browse/MHV-50831

image

## Screenshots

Before: 
image
image

After:
image
image

## Testing done

- Updated unit tests data + manual testing

## What areas of the site does it impact?

/my-health/medications/

## Acceptance criteria

- PDFs still say "Report generated by My HealtheVet and VA" in footer, should be "My HealtheVet on VA.gov"
-'Note:' and '6 months' need to be bolded
- Status definitions on PDF downloads have typo/formatting/text style issues. This one has a missing space in "you.But", wrong bullet style

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/platform/pdf/test/templates/medications/fixtures/medications_list.json [hunk 1]
```diff
@@ -39,7 +57,24 @@
...
+                    {
+                      "value": "Your VA provider prescribed this medication or supply to you. But we won’t send any shipments until you request to fill or refill it."
+                    },
+                    {
+                      "value": "We may use this status for either of these reasons:"
+                    },
+                    {
+                      "value": [
+                        "We’re not sure when you’ll need to fill this prescription, or",
+                        "You told us you have too much of this medication or supply"
```
Comment (agravell047) [inline src/platform/pdf/test/templates/medications/fixtures/medications_list.json]: @vmenshutin-bylight I'm assuming we don't need a period here?

File: src/applications/mhv/medications/util/constants.js [hunk 2]
```diff
@@ -30,40 +30,109 @@ export const dispStatusForRefillsLeft = [
...
+    {
+      value: `We put a hold on this prescription. You can’t request a refill until we remove the hold.`,
+    },
+    {
+      value: `We may use this status for either of these reasons:`,
+    },
+    {
+      value: [
+        `You told us you have too much of this medication or supply, or`,
+        `There’s a problem with this prescription`,
```
Comment (agravell047) [inline src/applications/mhv/medications/util/constants.js:73]: @vmenshutin-bylight I'm assuming we don't need a period here?

Comment (vmenshutin-bylight) [reply src/applications/mhv/medications/util/constants.js:73]: @agravell047 there was none before, it is a bulleted list

File: src/platform/pdf/test/templates/medications/fixtures/medications_list.json (hunk 1 above)
Comment (vmenshutin-bylight) [reply src/platform/pdf/test/templates/medications/fixtures/medications_list.json]: @agravell047 there was none before, it is a bulleted list

**Review by pjhill — APPROVED** [review]

**Pull request merged**